docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+/bin/sh
+
```

Comment:
probably want to get a different entry point here running the service

Reply:
i don't understand this comment

Reply:
by default should run the node code.

so should be something like:

```npm start```